### DISCUSSION TITLE
Project work

### DISCUSSION THREAD

**Discussion by shahid-25-khalid** [ocadotechnology/rapid-router#1580] [General]
Hi @lauracumming 

Could someone show me how to do the following:

1. Assign projects to yourself.
2. Open the specific code in VS Code or whatever code editor is preferred.
3. How to find the specific coding issue, so that I can start to test and fix it. 

I have a bit more time now to help with more issues and I am learning more programming (Java, C++, which go with my Python and R skills), so happy to use these skills. 

Cheers,
Shahid.

**Comment by shahid-25-khalid** [comment]
Hi @faucomte97  and @lauracumming,

I hope you're both well?
I have been promoted in my new Data Engineering role, so unfortunately do not have the time to help with any projects anymore. A real shame as I was enjoying learning about the Ocado work. If anything changes in the future and I have time again, I will let you know. 

I could recommend my cousin to the volunteering work at Ocado, as he is a first year software development student, and keen to gain some experience. 

Best wishes,
Shahid.

**Reply by lauracumming** [reply]
Congrats on the promotion @shahid-25-khalid, that's great news! Keep in touch.
